### ISSUE TITLE
Validation: if alternative conditions exist in a condition profile, should there be at least two?

### ISSUE THREAD

**Issue by mparsons-ce** [CredentialEngine/Schema-Development#664]
@stuartasutton @siuc-nate @jeannekitchens 
Some NavyCOOL credentials can have several options for requirements. I explained how to use alternativeCondition. 
I found that in some cases a credential would be published that included a condition profile with a single alternative condition.  It seems to me that if there is only one alternative condition, then it is not an alternative condition, it is the only condition. 
For publishing to the registry, I was going to add an edit:
- if alternative conditions are included in a condition profile, there must be at least two

Does that make sense?

**Comment by siuc-nate** [comment]
Can you tell from the content whether they mistakenly thought that the "alternative" was an alternative to the first (root) condition profile?

**Comment by jeannekitchens** [comment]
@mparsons-ce @siuc-nate this isn't a CTDL issue.  The basis should be the definition https://credreg.net/ctdl/terms#alternativeCondition.  Are you asking for Stuart's input based on the model and defintion?  Otherwise, is this something you two should get in a Zoom room or Slack thread and discuss?  Please remove this from the CTDL GitHub issues.

**Comment by mparsons-ce** [comment]
The purpose of the issue to confirm intent. The definition specifies there should be more than one alternative: 

> Constraints, prerequisites, entry conditions, or requirements in a context where **more than one alternative condition** or path has been defined and from which any one path fulfills the parent condition.

I was going to add an edit to the API to check for a minimum of two alternative conditions. If there is only one, then an error will be returned. 

Example from: 
https://sandbox.credentialengine.org/assistant/navy/graph/ce-11b49bcb-9f6a-44da-b77d-916bbb3c4dad
An education condition profile with two alternatives:
![image](https://user-images.githubusercontent.com/58825866/78040473-14581800-7335-11ea-84c1-bae73d6ae24a.png)

An experience condition profile, with a description, and one alternative condition:
![image](https://user-images.githubusercontent.com/58825866/78041149-fe972280-7335-11ea-8503-2b08080e3476.png)

Basically, this is a decision request: Should the API be updated to check:

> if a condition profile has alternative conditions, there must be at least two

**Comment by siuc-nate** [comment]
I think that makes sense.

**Comment by stuartasutton** [comment]
Yes, it makes sense to call a single entry for alternative to be an error.

**Comment by mparsons-ce** [comment]
Thanks, I have added an edit to require at least two items if AlternativeCondition has data.
